client/sections.js

```diff
@@ -112,9 +112,12 @@ const sections = [
 	},
 	{
 		name: 'plugins',
-		paths: [ '/plugins' ],
+		paths: [ '/plugins', `/([a-z]{2,3}|[a-z]{2}-[a-z]{2})/plugins` ],
```

Comment:
Now that we're starting to use those in more routes than `themes` ones, I think it's time to generalize them and reuse them, rather than redefine them for all localized routes.

Reply:
We could probably also use the magnificent locales from the start here and maybe avoid some unnecessary checks down the pipeline.

I'd also suggest we not block the PR with this, and address this in a follow-up.